app/ui/warehouses.py

```diff
@@ -214,14 +227,23 @@ def on_update_click_internal(self, *args) -> Optional[str]:
             return comment
         with connection.Connection.get() as conn:
             [i.update(conn, i) for i in new_warehouses]
+            # Twiddle the task state after a schedule has changed
+            update_task_state(conn)
 
     def list(self):
-        st.button(
-            "Clear State (TODO, for testing only)",
-            on_click=lambda: connection.execute(
-                f"delete from internal.{WarehouseSchedules.table_name}"
-            ),
-        )
+        if os.environ.get("OPSCENTER_LOCAL_DEV", False):
```

Comment:
I thikn we should still remove this before publishing.

Reply:
Agree. As Robert has started showing stuff off in "pre-release", I just wanted to consolidate some of my local hacks to hide this. Will go away in the end for sure.